examples/pubsub_realtime/pubsub_interrupt_publish.c

```diff
@@ -373,6 +372,7 @@ int main(void) {
     UA_StatusCode retval = UA_Server_run(server, &running);
 #if defined(PUBSUB_CONFIG_FASTPATH_FIXED_OFFSETS) || (PUBSUB_CONFIG_FASTPATH_STATIC_VALUES)
     if(staticValueSource != NULL) {
+        UA_DataValue_init(staticValueSource);
         UA_DataValue_delete(staticValueSource);
```

Comment:
Invalid free()... copied this line from [server_pubsub_publisher_rt_level.c#L210](https://github.com/open62541/open62541/blob/master/examples/pubsub/server_pubsub_publisher_rt_level.c#L210)